sample_app/app/views/users/new.html.erb

```diff
@@ -0,0 +1,49 @@
+<% provide :title, t(".signup") %>
+<h1><%= t ".signup" %></h1>
+<div class="row">
+  <div class="col-12">
+    <%= form_with(model: @user, local: true)  do |form| %>
+    <% if @user.errors.any? %>
```

Comment:
In ảnh @user.errors với trường hợp đăng ký thiếu ra đây cho anh?

Reply:
![Screenshot from 2020-08-04 11-40-11](https://user-images.githubusercontent.com/68525596/89258217-efacdd80-d651-11ea-8404-dde6a7b8c1ba.png)